docs/walkthrough.md

```diff
@@ -138,7 +138,16 @@ chmod +x ./kubectl
 We'll assume hereafter that all `kubectl` commands are using this
 newly-installed executable.
 
-To configure `kubectl` to communicate with the service catalog APU server:
+## Step 5 - Configuring `kubectl` to Talk to the API Server
+
+To configure `kubectl` to communicate with the service catalog API server, we'll have to
+get the IP address of the `Service` (and just the IP - no ports or `http://` prefixes)
+that sits in front of the API server pod(s). If you installed the catalog with one of the 
+`helm install` commands above, then this service will be called `catalog-catalog-apiserver`, 
```

Comment:
I'm ok with more text that tells them they need the IP of their "something" w/o the details of how to get it. Its just as it stands now it hints at using IPs that in some cases are wrong. Like, for me, the IP of the service, which is a private IP.  Even saying "make sure you don't try to use a private IP, you'll need to use...." would be good.